### ISSUE TITLE
Suggestions

### ISSUE THREAD

**Issue by ToniFabri** [victorrjimenezz/PS4-4PT#17]
Suggestions, could you use the default ps4 keyboard instead of white ? several apps use this keyboard because yours doesn't?

Are you going to put it in the background while you play or can you wait?

Another question the Ps4 can't reverse 100% of the internet by blocking, low in 50 or 70%, will this be possible?

thanks for the app, it's very good, keep updating

**Comment by victorrjimenezz** [comment]
> Suggestions, could you use the default ps4 keyboard instead of white ? several apps use this keyboard because yours doesn't?
> 
> Are you going to put it in the background while you play or can you wait?
> 
> Another question the Ps4 can't reverse 100% of the internet by blocking, low in 50 or 70%, will this be possible?
> 
> thanks for the app, it's very good, keep updating

Hi!! 

- Yes, I'm currently working on using the default keyboard to input text. The reason I had not added it earlier is because I could not manage to make it work, but now I'm close to getting it!

- Yes!! In the README file, where it says: `- [ ] Download pkgs in the background with DAEMONS`, that's exactly it, being able to download apps while playing or doing other stuff.

- Yes!! I'll add this in a future update

I'm happy that you like the app and appreciate the feedback :)

**Comment by ToniFabri** [comment]
But when starting the 4PT by downloading the game and after a game the 4PT needs to be closed, the download is active like this or stops ?

**Comment by victorrjimenezz** [comment]
> But when starting the 4PT by downloading the game and after a game the 4PT needs to be closed, the download is active like this or stops ?

If you close the 4PT Application, the download stops! However, as I mentioned, an option will be added in the future so that you do not need to close the app to play other games, and you'll be able to download games while playing/listening to music/watching movies...

**Comment by ToniFabri** [comment]
> > But when starting the 4PT by downloading the game and after a game the 4PT needs to be closed, the download is active like this or stops ?
> 
> If you close the 4PT Application, the download stops! However, as I mentioned, an option will be added in the future so that you do not need to close the app to play other games, and you'll be able to download games while playing/listening to music/watching movies...

I understand, If possible add remaining time finish download, sorry bad inglês i translate

**Comment by victorrjimenezz** [comment]
> > > But when starting the 4PT by downloading the game and after a game the 4PT needs to be closed, the download is active like this or stops ?
> > 
> > 
> > If you close the 4PT Application, the download stops! However, as I mentioned, an option will be added in the future so that you do not need to close the app to play other games, and you'll be able to download games while playing/listening to music/watching movies...
> 
> I understand, If possible add remaining time finish download, sorry bad inglês i translate

Yes, that’s a great idea, I’ll add it. Don’t worry about your english :)
